### PR TITLE
Tmma 461 expand cypress tests

### PR DESCRIPTION
**Pull request by Rodsy**

### CODE REVIEW

File: cypress/integration/large-dataset.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                .click()
+
+        // Goes to the ovid medline abstract file upload page
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Search Ovid MEDLINE®')
+
+            cy.get('input[type=file]')
+                .selectFile('tests/test-abstract-ovid-test-sample-5.txt')
```
Comment (asset-web) [inline cypress/integration/large-dataset.js:64]: @Rodsy Maybe consider using a larger test file.  NB: I think this sample is just the start of another file, likely to be part of the project.

If not the case, copy and tweak this test to create a new test, that processing a larger file.  NB: 5 records is not representative of normal usage.  Apologies this was not clear in the JIRA ticket.

Comment (Rodsy) [reply cypress/integration/large-dataset.js:64]: ok, cool.

File: cypress/integration/user-journey.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+            cy.get('table')
+                .eq(0)
+                .find('tr')
+                .eq(1)
+                .should('not.be.empty')
+
+        // now we are gonna wait a minute...
+
+            cy.wait(60000)
```
Comment (asset-web) [inline cypress/integration/user-journey.js:146]: @Rodsy Do both tests take 1 minute for the results to appear?  Reduce the wait time for the simple test where possible.

File: cypress/integration/user-journey.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+            cy.get('table')
+                .eq(0)
+                .find('tr')
+                .eq(1)
+                .should('not.be.empty')
+
+        // now we are gonna wait 30 seconds...
+
+            cy.wait(30000)
```
Comment (asset-web) [inline cypress/integration/user-journey.js:146]: @Rodsy Did you get the chance can you time this? If not can you try 15 seconds and then 7 seconds.  To find a consistent lowest value we can wait.   Alternatively rewrite the test to use the behaviour an end user would need to use, press reload and check in a loop.

Comment (Rodsy) [reply cypress/integration/user-journey.js:146]: ok, np boss.

**Comment by asset-web** [comment]
@Rodsy At present the user journey and large dataset tests need to be excluded running on py-web-t0.epi.bris.ac.uk which is why the Cypress tests failed, see email.  Can you exclude them from running as with the previous visualisation test please?

**Pull request merged**